backend-api/src/functions/asyncFinishBiometricSession/asyncFinishBiometricSessionHandler.ts

```diff
@@ -24,13 +163,43 @@ export async function lambdaHandlerConstructor(
   logger.addContext(context);
   logger.info(LogMessage.FINISH_BIOMETRIC_SESSION_STARTED);
 
-  const validateRequestBodyResult = validateRequestBody(event.body);
-  if (validateRequestBodyResult.isError) {
-    const errorMessage = validateRequestBodyResult.value.errorMessage;
+  const configResult = getFinishBiometricSessionConfig(dependencies.env);
+  if (configResult.isError) {
+    return serverErrorResponse;
+  }
```

Comment:
```suggestion
  const configResult = getFinishBiometricSessionConfig(dependencies.env);
  if (configResult.isError) {
    return serverErrorResponse;
  }
  
  const config = configResult.value
```

This is what we do in other lambdas. You can then access each envVar by doing `config.envVar` rather than doing `configResult.value.envVar`

I also add a comment where this can then be utilised